src/components/Home/Page1.jsx

```diff
@@ -194,17 +197,16 @@ class Page1 extends React.PureComponent {
                 playScale: [0, 1.65],
               }}
               location="page1-wrapper"
-            >
-              Feature
-            </Parallax>
+            ></Parallax>
           )}
           <h2>
-            What can <span>Groomers Express</span> do for you{' '}
+            What can <span>Express Groomers</span> do for you{' '}
           </h2>
-          <div className="title-line-wrapper page1-line">
-            <div className="title-line" />
-          </div>
           <OverPack>{children}</OverPack>
+          {/* render SearchForm - it just renders groomer cards right now */}
+          {/* <SearchForm /> */}
+          {/* render FormInput as a test to see if it will render correctly */}
+          {/* <FormInput /> */}
```

Comment:
not a major thing, doesn't require attention right now, but lets attempt to avoid leaving commented out code in our PR's